modules/browser/BrowserCapabilities.js

```diff
@@ -20,6 +20,7 @@ export default class BrowserCapabilities extends BrowserDetection {
         super();
         logger.info(
             `This appears to be ${this.getName()}, ver: ${this.getVersion()}`);
+        this.usesNewGumFlow() && logger.debug('Using the new GUM flow');
```

Comment:
seems like an odd place to log this, do we not have a log for this when we go down the new gum path?  if not i'd rather have it there than inside this util class

Reply:
Here is where we log the browser name, so I thought it was an appropriate place. I'm ok with dropping it altogether though, it was helpful when debugging.

Reply:
ok, my vote would be to log it in the flow that decides whether or not to use the new gum flow, but won't hold up the PR on the change.  just seems a bit weird to call out this one behavior of the browser capabilities in a log message here when there are lots of different flows we make based on the version.